js/index.js

```diff
@@ -1 +1,81 @@
 // Your code goes here
+//                  1
+let nav = document.querySelector('.nav');
+nav.addEventListener('dblclick', event => {
+    event.target.style.color = "blue";
+});
+//                  2
+nav.addEventListener('click', event => {
+    event.preventDefault();
+    event.target.style.color = "#212529";
+})
+//                  3 (not working)
+let signupBtns = document.querySelectorAll('.btn');
+for(let i = 0; i < signupBtns.length; i++) {
+    signupBtns[i].addEventListener('mouseover', event => {
+        event.target.style.transform = "scale(1.15)";
+    });
+};
+
+
+
+
+// signupBtns.addEventListener('mouseover', event => {
+//     event.target.style('transform', 'scale(1.15)');
+// });
+
+//                  4
+// let body = document.querySelector('body');
+// body.addEventListener('load', event => {
+//     event.target.setAttribute
+// })
+//                  5
+let botBoatImg = document.querySelector('.content-destination img')
+botBoatImg.addEventListener('load', event => {
+    event.target.style.border = "1px dashed red";
+});
+//                  6
+let footerP = document.querySelector('.footer p');
+footerP.addEventListener('mouseenter', event => {
+    event.target.style.fontSize = '2rem';
+});
+footerP.addEventListener('mouseleave', event => {
+    event.target.style.fontSize = '1.6rem';
+});
+//                  7
+let body = document.querySelector('body')
+body.addEventListener('keydown', event => {
+    event.target.style.backgroundColor = "purple";
+});
```

Comment:
So this right here is the makings of a crude keylogger 😈 You could check the value of that event, and add that to a string you are storing else where, and you will have a record of every keystroke that is fired in the body of your page .